airflow/security/kerberos.py

```diff
@@ -110,11 +112,11 @@ def detect_conf_var():
         return b'X-CACHECONF:' in f.read()
 
 
-def run():
-    if configuration.conf.get('kerberos', 'keytab') is None:
+def run(args):
+    if not args.keytab:
```

Comment:
This needs an `or configuration.conf.get('kerberos', 'keytab') is None` too

Reply:
no, it not needed, because we get configuration.conf.get('kerberos', 'keytab') as default value in arg - https://github.com/apache/incubator-airflow/blob/master/airflow/bin/cli.py#L1685 , so if args.keytabe will be None it mean what config empty too

Reply:
Ah I missed that, thanks.

Does it come out as None or `""`? (Oh shouldn't matter for `if not`)